docs/guest.md

```diff
@@ -22,24 +22,40 @@ The Data Manager Virtual Machine also needs to be installed to enable TKGS backu
 
 1. [Install Velero](https://velero.io/docs/v1.5/basic-install/)
 2. [Install Object Storage Plugin](#install-object-storage-plugin)
-3. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
+3. [Create Velero vSphere Plugin Config](#create-velero-vsphere-plugin-config)
+4. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
```

Comment:
Can the naming to consistent?
Either velero-plugin-for-vsphere or velero-vsphere-plugin